apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -104,6 +105,52 @@ describe('Delegation Controller', () => {
       });
   });
 
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid accountId', async () => {
+    const providerId = provider.msaId?.toString();
+    const { keypair } = users[1];
+    const invalidAccountId = `${keypair.address.slice(0, -1)}5`;
+    const getPath: string = `/v1/delegation/revokeDelegation/${invalidAccountId}/${providerId}`;
+    await request(httpServer).get(getPath).expect(400).expect({
+      statusCode: 400,
+      message: 'Invalid accountId',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with valid accountId: no msa', async () => {
+    const providerId = provider.msaId?.toString();
+    const keyring = new Keyring({ type: 'sr25519' });
+    const keypair = keyring.addFromUri('//Bob');
+
+    const getPath: string = `/v1/delegation/revokeDelegation/${keypair.address}/${providerId}`;
+    await request(httpServer).get(getPath).expect(404).expect({
+      statusCode: 404,
+      message: 'MSA ID for account not found',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid providerId', async () => {
+    const { keypair } = users[1];
+    const accountId = keypair.address;
+    const invalidProviderId = '9999';
+    const getPath: string = `/v1/delegation/revokeDelegation/${accountId}/${invalidProviderId}`;
+    await request(httpServer).get(getPath).expect(400).expect({
+      statusCode: 400,
+      message: 'Provider not found',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with no delegations', async () => {
+    const providerId = provider.msaId?.toString();
+    const keyring = new Keyring({ type: 'sr25519' });
+    const keypair = keyring.addFromUri('//Alice');
```

Comment:
`//Alice` is already our Provider, maybe use a different keypair here? If you want a registered MSA that's guaranteed not to have a current delegation to `//Alice`, you have 2 options:
1. Create a new keypair + MSA in the test
2. Revoke a delegation from one of the existing `users[]` in the test first.

Actually, both 1 & 2 should be separate test scenarios--(1) tests a `null` delegation, while (2) tests a revoked delegation.